src/main/java/com/cloudant/client/api/model/Response.java

```diff
@@ -64,6 +64,12 @@ public String getReason() {
         return response.getReason();
     }
 
+    /**
+     * @return reason code returned by the server or 0

```

Comment:
the comment says

>reason code

but the method is called `getStatusCode()`, I think the comment should say:

> the HTTP status code returned by the server or 0 if no code was available